### PR TITLE
added informational note about release order.

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by MichaelBaj — APPROVED** [review]

File: docs/about_releases.md [hunk 1]
```diff
@@ -3,6 +3,15 @@ title: List of Releases
 sidebar_label: 128T Releases
 ---
 
+:::info
+Issues resolved in a release are merged into subsequent releases chronologically AND lexicographically. 
+
+For example, issues resolved in `4.3.12`, which was released on 3/12/2021, are resolved in `4.5.6`, which was released on 3/26/2021 and also resolved in `5.1.0`, which was released on 3/15/2021, and so on.
+
+However, issues resolved in `4.3.12`, which was released on 3/12/2021 are not addressed in `4.5.5` because `4.5.5` was released on 2/10/2021, even though `4.5.5` is lexicographically higher than `4.3.12`.
```
Comment (jeffmart) [inline docs/about_releases.md:11]: Do we add to the end of this:
though `4.5.5` is lexicographically higher than `4.3.12`, `4.5.5` is chronologically older than `4.3.12`.

Yes it seems obvious but might be missed.

**Review by jeffmart — CHANGES_REQUESTED** [review]

File: docs/about_releases.md (hunk 1 above)
Comment (MichaelBaj) [reply docs/about_releases.md:11]: I think being blunt about this is important. I am in favor of adopting this addition.

**Review by MichaelBaj — DISMISSED** [review]

File: docs/about_releases.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Issues resolved in a release are merged into subsequent releases chronologically
 
 For example, issues resolved in `4.3.12`, which was released on 3/12/2021, are resolved in `4.5.6`, which was released on 3/26/2021 and also resolved in `5.1.0`, which was released on 3/15/2021, and so on.
 
-However, issues resolved in `4.3.12`, which was released on 3/12/2021 are not addressed in `4.5.5` because `4.5.5` was released on 2/10/2021, even though `4.5.5` is lexicographically higher than `4.3.12`.
+However, issues resolved in `4.3.12`, which was released on 3/12/2021 are not addressed in `4.5.5` because `4.5.5` was released on 2/10/2021, even though `4.5.5` is lexicographically higher than `4.3.12`, it is chronologically older than 4.3.12.
```
Comment (reidstidolph) [inline docs/about_releases.md:11]: This is a bit of a run on sentence. Consider breaking into two sentences?

**Review by reidstidolph — COMMENTED** [review]
Looks good, would just suggest breaking up the long sentence into shorter ones for better readability.

**Review by MichaelBaj — APPROVED** [review]

File: docs/about_releases.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/about_releases.md]: done

**Review by jeffmart — APPROVED** [review]

**Pull request merged**